crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -261,6 +272,11 @@ pub struct Config {
     pub max_rpc_connections: std::num::NonZeroU32,
 }
 
+pub struct WebSocket {
+    pub enabled: bool,
+    pub capacity: usize,
```

Comment:
Maybe it makes sense to make it `NonZeroUsize` or setting the limit to 0 for some subscription types is a valid config scenario?

Reply:
Aye sweet!